src/agent/agent.js

```diff
@@ -733,6 +734,8 @@ class Agent {
                     base_address: this.master_address,
                     certs: this._ssl_certs,
                     location_info: this.location_info,
+                    routing_hint: this.rpc.routing_hint,
```

Comment:
if I understand correctly the routing hint is what the agent thinks its location is (internal\external). what about the case of a cluster where we can have multiple noobaa server that each can be in a differnet location. do we know what to do when we'll get there?

Reply:
Currently we do not support this use case in the full. 
The current configuration is that agents are internal to the k8s that they ware installed in (and external to anybody else). In the future each location will get its own tag (instead of INTERNAL and EXTERNAL - the schema supports it) and the routing table will be calculated and send back to the client (the agent/endpoint) taking into account the client tag and the routing authority tag.